pkg/controllers/managementuser/rbac/handler_base.go

```diff
@@ -469,16 +469,17 @@ func (m *manager) ensureBindings(ns string, roles map[string]*v3.RoleTemplate, b
 	}
 
 	for key, rb := range desiredRBs {
-		logrus.Infof("Creating roleBinding %v", key)
 		switch roleBinding := rb.(type) {
 		case *rbacv1.RoleBinding:
+			logrus.Infof("Creating roleBinding %v", key)
```

Comment:
change this to debug?

Reply:
It was Info before, so I just kept it at the same level